catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -56,10 +56,11 @@
             <list>
                 <bean class="ddf.catalog.data.impl.types.TopicAttributes"/>
                 <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.LocationAttributes"/>
```

Comment:
❓ Is this needed? Looks like it's a dup of line 63

Reply:
reverted these changes